layers/+lang/ruby/layers.el

```diff
@@ -11,4 +11,4 @@
 
 (when (and (boundp 'ruby-backend)
            (eq ruby-backend 'lsp))
-  (configuration-layer/declare-layer-dependencies '(dap)))
+  (configuration-layer/declare-layer-dependencies '(dap prettier)))
```

Comment:
Hmmm thats not right dude, dap is an auxiliary layer for lsp server integration, however you want to have prettier required independently of lsp-mode, instead you need to check for your configuration variable and only require this layer if prettier is actually used.

Reply:
Ah of course, thanks. Updated.